src/pages/SchedulePage.js

```diff
@@ -1,119 +1,280 @@
+    schedule[dayToStr[dayNum]] !== undefined &&
+    Object.values(schedule[dayToStr[dayNum]]).map((course, i) => {
+      return <ScheduleCourse course={course} key={i} />;
+    });
 
   return (
-    <div style={{ padding: "2rem 10vw" }}>
-      <CourseCalendar>
-        <CalendarDay>
-          Monday
-          {_renderCourses(1, wishlist)}
-        </CalendarDay>
-        <CalendarDay>
-          Tuesday
-          {_renderCourses(2, wishlist)}
-        </CalendarDay>
-        <CalendarDay>
-          Wednesday
-          {_renderCourses(3, wishlist)}
-        </CalendarDay>
-        <CalendarDay>
-          Thursday
-          {_renderCourses(4, wishlist)}
-        </CalendarDay>
-        <CalendarDay>
-          Friday
-          {_renderCourses(5, wishlist)}
-        </CalendarDay>
-        <div>
-          <CalendarWeekend>
-            Saturday
-            {_renderCourses(6, wishlist)}
-          </CalendarWeekend>
-          <CalendarWeekend>
-            Sunday
-            {_renderCourses(0, wishlist)}
-          </CalendarWeekend>
-        </div>
-      </CourseCalendar>
-      {wishlist.length === 0 ? (
-        <span>No courses wishlisted yet!</span>
-      ) : (
-        <div>
-          <div
-            style={{ cursor: "pointer" }}
-            onClick={() => clearWishlist({ year, semester })}
-            onKeyPress={() => clearWishlist({ year, semester })}
-            role="button"
-            tabIndex={0}
-          >
-            Clear Wishlist
-          </div>
-          {}
-        </div>
-      )}
-    </div>
+    <Container>
+      <Calendar renderCourses={_renderCourses} />
+      <div
+        style={{
+          marginTop: "2rem",
+        }}
+      >
+        <Header>
+          <h2 className="wishlist">{`Wishlist (${wishlist.length})`}</h2>
+        </Header>
+        <WishlistCoursesList>
+          {wishlist.length === 0 ? (
+            <div
+              style={{
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "center",
+                padding: "10px",
+              }}
+            >
+              Wishlist displayed here
+            </div>
+          ) : (
+            wishlist.map((course, i) => {
+              return (
+                <WishlistCourse
+                  key={i}
+                  year={year}
+                  semester={semester}
+                  course={course}
+                  checkboxes={checkboxes}
+                  removeCourse={removeCourse}
+                  handleOnChange={handleOnChange}
+                />
+              );
+            })
+          )}
+        </WishlistCoursesList>
+        <ClearScheduleButton
+          style={{ cursor: "pointer", marginTop: "1rem", color: "#bd2f2f" }}
+          onClick={handleClearSchedule}
+          onKeyPress={() => clearSchedule({ year, semester })}
+          role="button"
+          tabIndex={0}
+        >
+          Clear Schedule
```

Comment:
Make this button a bit more visible. Possibly give it a thick border that is the same red as the text with some border radius. That would give it a more button-like appearance and make it feel less out of place